### PR TITLE
[BLOG] Add quansight-labs-work-update-for-september-2019

### PR DESCRIPTION
**Pull request by MarsBarLee**
Link to [blog post, successful build](https://labs-git-quansight-labs-work-update-for-septem-c61dac-quansight.vercel.app/blog/quansight-labs-work-update-for-september-2019).

## Links to other Quansight blogs:
There is no need to check for Blog2Blog links due to updated website build. 
## Text styling

- [x] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags in order to do so (with only one level-one header).
- [x] All links describe where they link to (for example, check the [Quansight labs website](https://labs.quansight.org/)).
- [x] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.

## Non-text contents

- [x] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing gifs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If it were to be read as plain text, the blog still makes sense and no information is missing.

### CODE REVIEW

File: apps/labs/posts/quansight-labs-work-update-for-september-2019.md [hunk 1]
```diff
@@ -0,0 +1,381 @@
...
+```
+
+## sphinx-math-dollar
+
+In SymPy, we make heavy use of LaTeX math in our documentation. For example,
+in our [special functions
+documentation](https://docs.sympy.org/dev/modules/functions/special.html#sympy.functions.special.hyper.hyper),
+most special functions are defined using a LaTeX formula, like ![The docs for besselj.](/images/besselj_docs.png)
+
+(from <https://docs.sympy.org/dev/modules/functions/special.html#sympy.functions.special.bessel.besselj>)
```
Comment (MarsBarLee) [inline apps/labs/posts/quansight-labs-work-update-for-september-2019.md:268]:
```suggestion
(from https://docs.sympy.org/dev/modules/functions/special.html#sympy.functions.special.bessel.besselj)
```

File: apps/labs/posts/quansight-labs-work-update-for-september-2019.md [hunk 2]
```diff
@@ -0,0 +1,381 @@
...
+  imageSrc: /posts/quansight-labs-work-update-for-september-2019/blog_hero_org.svg
+  imageAlt: 'An illustration of a light brown hand holding up a microphone, with some graphical elements highlighting the top of the microphone.'
+---
+
+As of November, 2018, I have been working at
+[Quansight](https://www.quansight.com/). Quansight is a new startup founded by
+the same people who started Anaconda, which aims to connect companies and open
+source communities, and offers consulting, training, support and mentoring
+services. I work under the heading of [Quansight
+Labs](https://www.quansight.com/labs). Quansight Labs is a public-benefit
```
Comment (noatamir) [inline apps/labs/posts/quansight-labs-work-update-for-september-2019.md:22]:
```suggestion
Labs](https://labs.quansight.org). Quansight Labs is a public-benefit
```

File: apps/labs/posts/quansight-labs-work-update-for-september-2019.md [hunk 3]
```diff
@@ -0,0 +1,381 @@
...
+is to improve the consistency of our docstrings in SymPy. She has already
+identified many key ways our docstring documentation can be improved, and is
+currently working on a style guide for writing docstrings. Some of the issues
+that Lauren has identified require improved tooling around the way the HTML
+documentation is built to fix. So some other SymPy developers and I have been
+working on improving this, so that she can focus on the technical writing
+aspects of our documentation.
+
+Lauren has created a draft style guide for documentation at
+https://github.com/sympy/sympy/wiki/SymPy-Documentation-Style-Guide. Please
```
Comment (noatamir) [inline apps/labs/posts/quansight-labs-work-update-for-september-2019.md:379]:
```suggestion
https://docs.sympy.org/latest/contributing/documentation-style-guide.html. Please
```

Comment (noatamir) [reply apps/labs/posts/quansight-labs-work-update-for-september-2019.md:379]: The wiki link doesn't work.  I replaced it with the documentation link.

File: apps/labs/posts/quansight-labs-work-update-for-september-2019.md [hunk 4]
```diff
@@ -0,0 +1,381 @@
+---
+title: 'Quansight Labs Work Update for September, 2019'
+published: October 7, 2020
+author: aaron-meurer
+description: 'In this post, I will detail some of the open source work that I have done
+recently, both as part of my open source consulting, and as part of my work on
+SymPy for Quansight Labs.'
+category:
```
Comment (noatamir) [inline apps/labs/posts/quansight-labs-work-update-for-september-2019.md:8]: [PyData ecosystem]

Comment (noatamir) [inline apps/labs/posts/quansight-labs-work-update-for-september-2019.md:8]:
```suggestion
category: [PyData ecosystem]
```

**Review by noatamir — APPROVED** [review]
LGTM

**Pull request merged**